semantics/psc-trees-binary.ads

```diff
@@ -150,6 +150,8 @@ package PSC.Trees.Binary is
       Right_Operand : Optional_Tree;
    end record;
 
+   type Tree_Ptr is access all Tree'Class;
```

Comment:
This declaration should probably be removed, since it is redundant with the similar declaration at line 84 of psc-trees.ads.